UPGRADE-5.3.md

```diff
@@ -38,6 +38,7 @@ FrameworkBundle
  * Deprecate the `session` service and the `SessionInterface` alias, use the `\Symfony\Component\HttpFoundation\Request::getSession()` or the new `\Symfony\Component\HttpFoundation\RequestStack::getSession()` methods instead
  * Deprecate the `KernelTestCase::$container` property, use `KernelTestCase::getContainer()` instead
  * Rename the container parameter `profiler_listener.only_master_requests` to `profiler_listener.only_main_requests`
+ * Deprecate option `--xliff-version` of the `translation:update` command, use e.g. `--output-format=xlf20` instead
```

Comment:
I have not checked, but I think we are using `--format` in other commands.

Reply:
we are using output-format in TranslationUpdateCommand...